fastlane/Fastfile

```diff
@@ -149,8 +160,17 @@ platform :android do
     )
   end
 
+  desc "Tag current branch with current version number"
+  lane :tag_current_branch do |options|
+    version_number = current_version_number
+    check_no_git_tag_exists(version_number)
```

Comment:
Mentioned it in the other PR, but we need to bring in this method I think

Reply:
oh great catch, I had the same issue in another repo before. Is there any reason why this is not an action in the common plugin?

Reply:
Well... I guess it's a pretty simple method, so the action would be equally simple... But if it's going to be reused a lot, it might make sense to create an action for it.

Reply:
Yes, you're right, it's super simple. Let's just copy paste it for now so I can wrap up the sprint with Android, Flutter and React Native Purchases release trains. Cordova and Unity still need deployment jobs.